src/util/getExperienceName.ts

```diff
@@ -0,0 +1,41 @@
+import { BlobServiceClient, ContainerClient } from "@azure/storage-blob";
+
+const sasToken =
+  "?sv=2020-08-04&ss=b&srt=sco&sp=rwdlacitfx&se=2022-09-30T20:46:30Z&st=2021-11-30T13:46:30Z&spr=https&sig=e3MgX4FkExWFc5LWFv5RGZdLdyzSMy3ZSSX76%2BPzGFs%3D";
+
+//Store the name of the experience
+var ExperienceName: string = "Experience";
+
+//Set the name of the experience
+function setExperienceName(name: string) {
+  ExperienceName=name;
+  console.log(ExperienceName);
+}
+
+//Get the name of the experience
+function getExperienceName() {
+  console.log(ExperienceName);
+  return ExperienceName;
+}
+
+const getBlobsInContainer = async (userId) => {
+  const returnedBlobUrls: string[] = [];
+
+  // get BlobService = notice `?` is pulled out of sasToken - if created in Azure portal
+  const blobService = new BlobServiceClient(
+    `https://addressabletest1.blob.core.windows.net/?${sasToken}`
+  );
+
+  // get Container - full public read access
+  const containerClient: ContainerClient =
```

Comment:
this line on it's own will cause the rest error because if the container does not exist, it still tries to retreive the blobs from that container. the solution is either use this line of code to ensure the container is created if it does not exist already : 
  await containerClient.createIfNotExists({
    access: "container",
  });
Another solution is to create a container once the user creates an account but that might come furthur in development because the current login page serves as create account as well.

Reply:
This resolved the runtime issue with new users! Thank you very much! :)